pom.xml

```diff
@@ -7,7 +7,7 @@
   <properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <graalvm.version>21.2.0</graalvm.version>
+    <graalvm.version>21.3.0</graalvm.version>
```

Comment:
The plugin `org.graalvm.nativeimage:native-image-maven-plugin` doesn't have 21.3 (yet?).

<img width="798" alt="Screen Shot 2021-11-02 at 6 08 25 PM" src="https://user-images.githubusercontent.com/28604/139958579-6119dc6a-9ac6-41a4-8f92-3bdc2fad0b60.png">

Reply:
Hmm, yes. Seems like they did not publish a version `21.3.0` for the plugin which is odd. I will have to do some research. I remember something about them mentioning that they changed the artifact name/location of this plugin.

Reply:
I think the community is moving towards a plugin defined in a new location:

```
          <plugin>
            <groupId>org.graalvm.buildtools</groupId>
            <artifactId>native-maven-plugin</artifactId>
            <version>${native.maven.plugin.version}</version>
            ...
          </plugin>
```

See for change notes: https://graalvm.github.io/native-build-tools/0.9.7.1/maven-plugin.html

Reply:
The updated plugin definition should look something like this:

```
​<profile>
     ​<id>native</id>
     ​<build>
       ​<plugins>
         ​<plugin>
           ​<groupId>org.graalvm.buildtools</groupId>
           ​<artifactId>native-maven-plugin</artifactId>
           ​<version>${native.maven.plugin.version}</version>
           ​<extensions>true</extensions>
           ​<executions>
             ​<execution>
               ​<id>build-native</id>
               ​<goals>
                 ​<goal>build</goal>
               ​</goals>
               ​<phase>package</phase>
             ​</execution>
           ​</executions>
           ​<configuration>
           ...
           ​</configuration>
         ​</plugin>
       ​</plugins>
     ​</build>
   ​</profile>
```

Reply:
Thank you for the pointer. Let me try it.

Reply:
Ah, I think they changed the plugin executions and configuration too. You will have to follow this model: https://graalvm.github.io/native-build-tools/0.9.7.1/maven-plugin.html

Let me know if this is too tedious; I can do the migration if you would prefer. The plugin config actually has to be changed for each sample to migrate to the new plugin.

Reply:
No, I take this. Good opportunity to learn GraalVM. (Indeed, I just installed native-image command)